lib/controller/index.js

```diff
@@ -110,12 +114,15 @@ let Controller = BaseController;
 Controller = require('./mixins/resolve-path')(Controller);
 Controller = require('./mixins/base-url')(Controller);
 Controller = require('./mixins/translate')(Controller);
+Controller = require('./mixins/journey-model')(Controller);
 Controller = require('./mixins/session-model')(Controller);
 Controller = require('./mixins/check-session')(Controller);
 Controller = require('./mixins/check-progress')(Controller);
 Controller = require('./mixins/csrf')(Controller);
 Controller = require('./mixins/invalidate-fields')(Controller);
 Controller = require('./mixins/back-links')(Controller);
+Controller = require('./mixins/next-step')(Controller);
+Controller = require('./mixins/edit-step')(Controller);
```

Comment:
We also need to check that the edit-step mixin is added after the back-links mixin and the next-step mixin because it relies on their super methods.